internal/pkg/service/common/etcdop/watch.go

```diff
@@ -11,7 +11,7 @@ import (
 type EventT[T any] struct {
 	*etcd.Event
 	Header *etcdserverpb.ResponseHeader
-	Value  *T
+	Value  T
```

Comment:
Vsetky operacie nad prefixom vratia `T`, tak som  to upravil aj tu.
Ci je `T` prazdne je jasne z typu eventu, ... bude to prazdna hodnota iba pri `Delete` evente.